app/dashboard/views.py

```diff
@@ -4485,12 +4499,21 @@ def hackathon_save_project(request):
 @login_required
 def get_project(request, project_id):
     profile = request.user.profile if request.user.is_authenticated and hasattr(request.user, 'profile') else None
+    params = project(project_id)
+    if not params:
+        raise Http404("The project doesnt exists.")
+
+    return JsonResponse(params)
+
 
+def project(project_id):
     project = HackathonProject.objects.filter(pk=project_id).nocache().first()
     if not project:
-        raise Http404("The project doesnt exists.")
+        raise None
```

Comment:
^ why are we changing this ?

Reply:
@thelostone-mc cause `project` isn't a view function. it's used in `get_project` view @ line 4502